arm-resources/subscriptions/2016-06-01/swagger/subscriptions.json

```diff
@@ -0,0 +1,312 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SubscriptionClient",
+    "version": "2016-06-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
```

Comment:
does the service actually support text/json in produces and consumes? When we tested this aspect for one of the services, it returned a 400 bad request with "text/json". So we wanted to confirm.